pkg/repro/repro.go

```diff
@@ -41,6 +41,14 @@ type instance struct {
 	executorBin string
 }
 
+func reverseEntries(entries []*prog.LogEntry) []*prog.LogEntry {
```

Comment:
Move this closer to the bottom of the file. This is not C which requires function declarations, so people used to put the least important stuff first.